docs/faq.rst

```diff
@@ -230,8 +230,8 @@ Transaction Model and Locking`_ in the MySQL user manual.
 The worker isn't doing anything, just hanging
 ---------------------------------------------
 
-**Answer:** See `MySQL is throwing deadlock errors, what can I do?`_.
-            or `Why is Task.delay/apply\* just hanging?`.
+**Answer:** See `MySQL is throwing deadlock errors, what can I do?`_,
+or `Why is Task.delay/apply\*/the worker just hanging?`_.
```

Comment:
This is a reference to another question, which is not aligned with the actual question.

If you look two questions below this answer, you'll see it is indeed:

> Why is Task.delay/apply*/the worker just hanging?

and not 

> Why is Task.delay/apply* just hanging?